src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodRecordsAndCompactCtors.java

```diff
@@ -62,7 +62,7 @@ public record MySecondRecord() {
     }
 
     // Record component, compact ctor
-    public record MyThirdRecord(String myString) {
+    public record MyThirdRecord(String myString) { // violation 'Expected @param tag for 'myString''
```

Comment:
Are these type of violations new? It doesn't really make sense to me for it be here where this is not the Javadoc, and Javadoc is on the constructor.